### DISCUSSION TITLE
Problems when primary color is red

### DISCUSSION THREAD

**Discussion by rasco** [directus/directus#15658] [Feature Requests]
Not really important, but this is confusing to users:
When I make an "info" notice field, it uses our project's primary color.
But since our primary color is red, it looks like an error! 
See screenshot attached.

Maybe change the "info" color to green, regardless of the custom color scheme?

---

The same problem also arises with the "Status" dot: 
The default color for the "published" status is the primary color.
But when it's red it looks like it is offline, although it is actually published.
So currently I'm changing the status dots' "published" color to green every time I need them

---

<img width="927" alt="Screenshot 2022-09-20 at 1 06 51 PM" src="https://user-images.githubusercontent.com/289536/191172166-9fcb9c37-698c-4f0b-b55f-390807aab908.png">

**Comment by azrikahar** [comment]
FWIW there is a warning regarding the usage of red hues in the docs [over here](https://docs.directus.io/extensions/themes.html#example) (might still not be as visible compared to putting this in Project Settings -> Custom CSS):

![image](https://user-images.githubusercontent.com/42867097/191188347-1de8a2eb-acb5-4cf8-952f-a50b70ebc463.png)

but I do acknowledge that there is a case to make this "info styled with primary color" as an opt-in thing rather than the current opt-out strategy. Perhaps once this is finalized, it can be looked into along with the notice improvements mentioned here: https://github.com/directus/directus/discussions/13475#discussioncomment-2801677. On a broader theming/branding sense, this can also be something to do with #13247.

In the mean time, you should be able to target the styling for "info" type notice with the following selector in your Custom CSS config:

```css
#app .v-notice.info {
    --v-notice-icon-color: green;
    --v-notice-border-color: green;
}
```

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
